test/hist.jl

```diff
@@ -126,10 +121,10 @@ end
 @testset "Histogram show" begin
     # hist show
     show_h = sprint(show, fit(Histogram,[0,1,2], closed=:left))  # FIXME: closed
-    @test contains(show_h, "edges:\n  0.0:1.0:3.0")
-    @test contains(show_h, "weights: $([1,1,1])")
-    @test contains(show_h, "closed: left")
-    @test contains(show_h, "isdensity: false")
+    @test Compat.occursin("edges:\n  0.0:1.0:3.0", show_h)
```

Comment:
Can drop the `Compat.` here too

Reply:
There are a few other places where the prefix isn't needed either. And sometimes `using Compat: ...` would make sense instead of repeating `Compat.`.